src/Polly.Core/CircuitBreaker/CircuitBreakerPredicateArguments.cs

```diff
@@ -3,4 +3,7 @@ namespace Polly.CircuitBreaker;
 /// <summary>
 /// Arguments used by <see cref="CircuitBreakerStrategyOptions{TResult}.ShouldHandle"/> predicate.
 /// </summary>
+/// <remarks>
+/// Always use constructor when creating this struct, otherwise we do not guarantee the binary compatibility.
```

Comment:
For all the usages:

```suggestion
/// Always use the constructor when creating this struct, otherwise we do not guarantee binary compatibility.
```